### DISCUSSION TITLE
Multilanguage support

### DISCUSSION THREAD

**Discussion by fedetango** [dailydotdev/daily#1229] [Feature Requests]
Please add multilanguage support for posts.

There is a large part of the public that speaks or prefers to speak other languages. This feature would increase the reach of Daily and the posts.

I think the ideal way to do this would be to have each user choose the languages ​​they want to read in their profile.

Each post should have the possibility of presenting several versions of itself in different languages

Thank you!

**Comment by idoshamun** [comment]
Hi @fedetango,

Thanks for the suggestion. Currently localization it's out of our scope.
Most of the content on the platform currently is in English.
I know some members created localized squads in non-English languages
